### ISSUE TITLE
show all the nodes(expanded form) in the space tree by default

### ISSUE THREAD

**Issue by deepumon** [philogb/jit#160]
how can i show all the nodes from the space tree in an expanded state by default? Can u give me a solution? Is it related with LevelsToShow?

**Comment by dilip-vishwa** [comment]
Yes, you can specify LevelsToShow=100 or any higher no. you may think of.
